tw-NEDa-backend/src/main/java/org/neda/EarthquakeController.java

```diff
@@ -6,17 +6,24 @@
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.web.bind.annotation.RequestMapping;
 import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.bind.annotation.ResponseBody;
 import org.springframework.web.bind.annotation.RestController;
 
 import java.sql.Date;
 
 @RestController
 public class EarthquakeController {
 
-    @Autowired
     private EarthquakeRepository earthquakeRepository;
 
-    @RequestMapping(value = "/works/earthquake", method = RequestMethod.GET)
+
+    @Autowired
+    public EarthquakeController(EarthquakeRepository earthquakeRepository){
+        this.earthquakeRepository = earthquakeRepository;
+    }
+
+    @RequestMapping(value = "/earthquake", method = RequestMethod.GET)
```

Comment:
GET method is used primary for read operations.
You should not create entities in the db when such call is made.

This should be changed to a POST request in which you will later add the values that you've hardcoded in the method.